src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/IntegrasjonerConfig.kt

```diff
@@ -45,6 +45,10 @@ class IntegrasjonerConfig(@Value("\${FAMILIE_INTEGRASJONER_URL}") private val in
 
     val navKontorUri: URI = UriComponentsBuilder.fromUri(integrasjonUri).pathSegment(PATH_NAV_KONTOR).build().toUri()
 
+    val arbeidsfordelingOppfølgingUri: URI = UriComponentsBuilder.fromUri(integrasjonUri).pathSegment(
```

Comment:
Hva er forskjellen på dette endepunktet og den med relasjoner?

Reply:
Her har vi kanskje et "hull" mtp relasjonsbiten - det kan iallefall diskuteres. 

Her er vi interessert i å finne dét lokalkontoret som passer for brukeren/personen. Siden dette er geografisk knyttet blir det potensielt pussig å finne den med "høyest sikkerhetsnivå" - og finne lokalkontor deretter?